src/main/java/edu/cornell/kfs/coa/batch/service/impl/CreateClosedLaborAccountsCsvServiceImpl.java

```diff
@@ -0,0 +1,253 @@
+                fromToRange.put(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FROM_DATE, sqlComputedFromDate);
+                fromToRange.put(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.TO_DATE, getDateTimeService().getCurrentSqlDate());
+                LOG.info("getClosedLaborAccountsDateRange: Periodic file will have FROM_DATE = " + fromToRange.get(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FROM_DATE) 
+                    + " and a TO_DATE = " + fromToRange.get(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.TO_DATE)); 
+            } else {
+                LOG.info("getClosedLaborAccountsDateRange: Periodic file will NOT be created.");
+            }
+            
+        } else {
+            LOG.error("getClosedLaborAccountsDateRange: Unknown Type of Closed Account File to Create = " + typeOfClosedAccountFileToCreate.get(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPE_IS));
+        }
+        return fromToRange;
+    }
+    
+    private Date getJavaSqlDateForComputedDate(int daysPrevious) {
+        LocalDate computedFromDate = computeDateBeingDaysPreviousFromToday(daysPrevious);
+        java.sql.Date sqlDate;
+        try {
+            sqlDate = getDateTimeService().convertToSqlDate(computedFromDate.toString());
+        } catch (ParseException pe) {
+            LOG.error("getJavaSqlDateForComputedDate: ParseException caught for computedFromDate = " + computedFromDate.toString());
+            sqlDate = null;
+        }
+        return sqlDate;
+    }
+    
+    private LocalDate computeDateBeingDaysPreviousFromToday(int daysPrevious) {
+        LOG.info("computeDateBeingDaysPreviousFromToday: daysPrevious = " + daysPrevious);
+        DateTime todayAsDateTime = new DateTime(getDateTimeService().getCurrentDate());
+        DateTime datePreviousAsDateTime = todayAsDateTime.minusDays(daysPrevious);
+        LOG.info("computeDateBeingDaysPreviousFromToday: datePreviousAsDateTime = " + datePreviousAsDateTime.toLocalDate().toString());
+        return datePreviousAsDateTime.toLocalDate();
+    }
+    
+    private void writeClosedLaborAccountsToCsvFormattedFile(Map<String, Date> fileDataRangeSpecified) {
+        LOG.info("writeClosedLaborAccountsToCsvFormattedFile: Prior to ensuring directory to hold outbound file exists.");
+        ensureOutbundDirectoryExists();
+        
+        LOG.info("writeClosedLaborAccountsToCsvFormattedFile: Prior to obtaining data with SQL query for date range. This may take a while.");
+        List<LaborClosedAccount> laborClosedAccountsData = getClosedLaborAccountsByDateRangeDao().obtainLaborClosedAccountsDataFor(fileDataRangeSpecified);
+        
+        LOG.info("writeClosedLaborAccountsToCsvFormattedFile: Have the data. Will now write it out.");
+        writeClosedAccountsToCsvFile(laborClosedAccountsData);
+    }
+    
+    private void ensureOutbundDirectoryExists() {
```

Comment:
I think if you configure the output directory in Spring, it will create the directory for us.  If so, this function may not be needed

Reply:
Confirmed directory is created when originally not present.  Removing method.